### DISCUSSION TITLE
thanhle.blog/mot-so-suy-nghi-ve-luong

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#64] [Ideas]
# thanhle.blog/mot-so-suy-nghi-ve-luong

Lương giống như Se* vậy, hiểu hơn để không ngại nói về nó :>

https://thanhle.blog/blog/mot-so-suy-nghi-ve-luong

**Comment by thanhlmm** [comment]
A nice viz about how ppl see you vs how you see your work

![image](https://user-images.githubusercontent.com/9281080/195998830-1e60d7b2-3279-4153-8978-a723afc7fcfc.png)

**Comment by thanhlmm** [comment]
Tham khảo thêm báo cáo IT để có bức tranh toàn cảnh hơn https://topdev.vn/page/bao-cao-it-viet-nam

**Comment by thanhlmm** [comment]
https://viblo.asia/p/nam-moi-noi-chuyen-ve-luong-thuong-zOQJwQmNVMP

Một góc nhìn khác về lương!
